2021-10-28 07:24:24,078 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-10-28 07:24:24,674 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57574)
2021-10-28 07:24:24,674 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1635395064.6742558, 'user': {'account_name': 'test0'}}
2021-10-28 07:24:25,088 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57576)
2021-10-28 07:24:25,589 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1635395065.0882614, 'user': {'account_name': 'test1'}}
2021-10-28 07:25:15,706 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test1', 'time': 1635395115.4453285, 'mess_text': 'hi'}
2021-10-28 07:25:15,710 INFO server.py Отправлено сообщение пользователю test1 от пользователя test1.
2021-10-28 07:25:59,308 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1635395158.8724616, 'mess_text': 'hi'}
2021-10-28 07:25:59,309 ERROR server.py Пользователь test2 не зарегистрирован на сервере, отправка сообщения невозможна.
2021-10-28 07:26:25,874 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test0', 'to': 'test1', 'time': 1635395185.8137999, 'mess_text': 'hi'}
2021-10-28 07:26:25,878 INFO server.py Отправлено сообщение пользователю test1 от пользователя test0.
2021-10-28 07:26:48,922 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1635395208.6229005, 'account_name': 'test0'}
2021-10-28 07:26:53,434 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1635395213.3805115, 'account_name': 'test1'}
